src/presets/svelte-transition.ts

```diff
@@ -0,0 +1,7 @@
+import { ImportsMap } from '../types'
+
+import { SvelteTransitionAPI } from './svelte'
+
+export default <ImportsMap>({
+  'svelte/transition': SvelteTransitionAPI,
```

Comment:
we could simply have them in one file

Reply:
so, remove all single entries?

Reply:
should I remove the animate, easing... entries from on the default export?

Reply:
yes, just have `presets/svelte` with named exports will do.

Reply:
done